### PR TITLE
LLDB: Print actual node structure  [ci skip]

### PR DESCRIPTION
**Pull request by yui-knk**

### CODE REVIEW

File: misc/lldb_rb/utils.py [hunk 1]
```diff
@@ -249,7 +249,224 @@ def inspect(self, val):
...
+                elif nd_type == self.ruby_globals["NODE_RIPPER"]:
+                    self._append_command_output("p *(struct RNode_RIPPER *) %0#x" % val.GetValueAsUnsigned())
+                elif nd_type == self.ruby_globals["NODE_RIPPER_VALUES"]:
+                    self._append_command_output("p *(struct RNode_RIPPER_VALUES *) %0#x" % val.GetValueAsUnsigned())
+
+                # Internal nodes
+                elif nd_type == self.ruby_globals["NODE_DEF_TEMP"]:
+                    self._append_command_output("p *(struct RNode_DEF_TEMP *) %0#x" % val.GetValueAsUnsigned())
+                elif nd_type == self.ruby_globals["NODE_EXITS"]:
+                    self._append_command_output("p *(struct RNode_EXITS *) %0#x" % val.GetValueAsUnsigned())
```
Comment (nobu) [inline misc/lldb_rb/utils.py:466]: I suspect these symbols do not appear in `self.ruby_globals`.

Comment (yui-knk) [reply misc/lldb_rb/utils.py:466]: Oh, thank you.
One idea is moving `enum internal_node_type` into `internal/ruby_parser.h` and defining `enum internal_node_type     internal_node_type;` in `debug.c`.
Or removing internal nodes from `utils.py`.
At the moment, I don't have strong opinion. Do you have any preference?

Comment (nobu) [reply misc/lldb_rb/utils.py:466]: For the time being, how about ignoring these internal nodes?
Regarding `NODE_DEF_TEMP` at least, I'm planning to remove cast it to `NODE`.

Comment (yui-knk) [reply misc/lldb_rb/utils.py]: Make sense, updated to remove internal nodes.

**Review by nobu — APPROVED** [review]

**Review by nobu — APPROVED** [review]

**Pull request merged**